beesdoo_pos/__manifest__.py

```diff
@@ -12,7 +13,7 @@
     "author": "Beescoop - Cellule IT, Coop IT Easy SCRLfs",
     "website": "https://github.com/beescoop/Obeesdoo",
     "category": "Point Of Sale",
-    "version": "12.0.1.0.0",
+    "version": "12.0.1.0.1",
```

Comment:
I suggest to increment the second digit of the version number as it adds new functionality in the module and not a fix in the current one. See https://semver.org
```suggestion
    "version": "12.0.1.1.0",
```

Reply:
done :)